sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/SpanProcessor.java

```diff
@@ -56,13 +56,17 @@
    *
    * @return a {@link CompletableResultCode} which completes when shutdown is finished.
    */
-  CompletableResultCode shutdown();
+  default CompletableResultCode shutdown() {
+    return CompletableResultCode.ofSuccess();
```

Comment:
```suggestion
    return forceFlush();
```

Reply:
Can you explain why do we need this? : ）

Reply:
Because shutdown should normally ensure that everything is flushed. See https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#shutdown-1

> Shutdown MUST include the effects of ForceFlush.

Reply:
👍 Thanks. fixed.